### ISSUE TITLE
Autopdater not avaialble

### ISSUE THREAD

**Issue by GliderHacker** [jaredballou/insurgency-sourcemod#28]
Hi Jared...I tried to install the Updater plugin in my Sourcemod folder, Insurgency game.
Basic Sourcemod works fine, I can perform sm admin command into the server or client console.
So, I have installed the files in the proper folders, changed the map...but the plugin is not avaialble and sm_updater_status is an unknown command.
What am I missing?
Thanks

**Comment by GliderHacker** [comment]
More info...
the error log file says:
`L 08/27/2016 - 18:22:03: [SM] Exception reported: This plugin requires either the cURL, Socket, or SteamTools extension to function.`

But cURL.inc, socket.inc and steamtools.inc are uploaded into the sourcemod/scripting/include folder ...

**Comment by GliderHacker** [comment]
Oh...maybe I fixed the issue: following the error log files I have installed into the server extensions the libraries curl.ext.dll, curl.ext.so, socket.ext.dll and socket.ext.so ( no SteamTools extension found ind the downloaded package ) and now the plugin seems working fine.
Is this correct? Since I don't see any info about that in this page about plugin installation.
Thanks

**Comment by jaredballou** [comment]
Yeah. The updater plugin is actually made by GoD-Tony on the forums. I just ported it to SM 1.8, and added a few more features. The extensions were the issue, as you expected. I am going to be pushing a new version of my updater later this week, which will reduce the debug messages significantly.
